marketplaceItems/appreciate/appResources/appData/resources/css/style.css

```diff
@@ -0,0 +1,260 @@
+*, *:before, *:after {
+    -webkit-box-sizing: inherit;
```

Comment:
Are these necessary since we have only one standard browser?

Reply:
(or is it different on Mac/Linux?)

Reply:
these really aren't necessary, but Lint will complain if we only use `-webkit-box-sizing` (which is the one Chrome cares most about). it's also pretty good webdev practice to understand how this CSS affects other browsers, even if we aren't using them. additionally, if we want to test our content in-browser (which is good practice), we should make sure we are going to see the same thing in hifi's Chromium-based browser.